ote/src/cljc/ote/transit_changes.cljc

```diff
@@ -192,18 +192,19 @@
                 [l r (trip-stop-differences l r)])
               combined-trips)))))
 
-#_(defn week=
+
+(defn week=
```

Comment:
Please consider renaming because function is not a genuine comparison but implements logic.
E.g.
`traffic-wk-change?` and put `not`s where it's called 
or alternatively rename to `traffic-wk-same?`
or something better :)